<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailTransport::TransportManagementWidget</class>
 <widget class="QWidget" name="MailTransport::TransportManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <widget class="TransportListView" name="transportList">
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KPushButton" name="addButton">
     <property name="text">
      <string>A&amp;dd...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="editButton">
     <property name="text">
      <string>&amp;Modify...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KPushButton" name="renameButton">
     <property name="text">
      <string>&amp;Rename</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KPushButton" name="removeButton">
     <property name="text">
      <string>Remo&amp;ve</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KPushButton" name="defaultButton">
     <property name="text">
      <string>&amp;Set as Default</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TransportListView</class>
   <extends>QTreeWidget</extends>
   <header location="global">transportlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
